src/utils/datastructs.jl

```diff
@@ -29,17 +29,17 @@ struct OptVariable{T,N,Ax,L<:NTuple{N,Dict}} <: AbstractArray{T,N}
 end
 
 """
-      OptResult{status::Symbol,objective::Float64,variables::Dict{String,Any},sets::Dict{String,Array},opt_config::Dict{String,Any},opt_info::Dict{String,Any}}
+      OptResult{status::Symbol,objective::Number,variables::Dict{String,Any},sets::Dict{String,Array},opt_config::Dict{String,Any},opt_info::Dict{String,Any}}
```

Comment:
Are you aware of a generic Float type similar to `Int`? 
I've always liked to use Float64 instead of number because it can catch input errors more easily, which with number wouldn't be the case anymore.